docker-compose.yml

```diff
@@ -41,3 +41,14 @@ services:
     image: rabbitmq:3.7-management
     restart: on-failure
     network_mode: host
+
+  prometheus:
+    image: prom/prometheus
+    restart: on-failure
+    network_mode: host
+    volumes:
+      - ./prometheus.yml:/etc/prometheus/prometheus.yml
```

Comment:
I think it's good idea to mov prometheus.yml from the main dir, maybe `dev` dir?

Reply:
done